server/api/scheduler.go

```diff
@@ -325,5 +325,5 @@ func (h *schedulerConfigHandler) GetSchedulerConfig(w http.ResponseWriter, r *ht
 		sh.ServeHTTP(w, r)
 		return
 	}
-	h.rd.JSON(w, http.StatusNotAcceptable, errNoImplement.Error())
```

Comment:
The condition to process this is that RaftCluster is nil, so it was wrong before.